sonar-kotlin-plugin/src/main/resources/static/documentation.md

```diff
@@ -12,6 +12,22 @@ key: kotlin
 
 You can discover and update Kotlin-specific [properties](/analysis/analysis-parameters/) in:  <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Kotlin](/#sonarqube-admin#/admin/settings?category=kotlin)**.
 
+## Kotlin Analysis and Bytecode
+If you are not using the Maven or Gradle Sonar plugins, it is strongly recommended to provide the paths of all dependency binaries used by
+the project in order to improve the analysis accuracy. You can provide these using the `sonar.java.libraries` property. This is a list of 
+comma-separated paths to files with third-party libraries (JAR or Zip files) used by your project. Wildcards can be used: 
+`sonar.java.libraries=path/to/Library.jar,directory/**/*.jar`
+
+Note that if you use the Gradle or Maven Sonar plugins to scan your code, these plugins will usually auto-detect the value for the property.
```

Comment:
Note that if you use the "SonarScanner for Maven" or "SonarScanner for Gradle" to scan your code, these scanners will auto-detect the value for the property. Thus you don't need to provide them.